### PR TITLE
TAM Request

### PR DESCRIPTION
**Pull request by justinorringer**
```
Your input ->  i am a tam
I can help make a custom access request.
Please provide the customer's Account ID.
Your input ->  123
What is their Org ID?
Your input ->  123
? How long do you need access for? Select a preset below. 1: 3 days (3 days)
This request will grant you access to all roles in the customer account. For more granular access, use the form [here](/internal/access-requests).
? Are you sure you want to request access to customer account 123? 1: Yes (/intent_core_yes)
Your request for access to customer account 123 has been submitted. The customer will be notified shortly.
Future requests can also be made using the form [here](/internal/access-requests).
```

[Doc on the RBAC endpoint](https://spaces.redhat.com/pages/viewpage.action?pageId=472848321)

### CODE REVIEW

File: actions/platform/tam/setup_actions.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class ActionAccessRequestTAMReset(Action):
+    def name(self) -> Text:
+        return "action_access_request_tam_reset"
+
+    async def run(
+        self, dispatcher: CollectingDispatcher, tracker: Tracker, domain: Dict
+    ) -> List[Dict[Text, Any]]:
+        print("Resetting TAM slots")
```
Comment (josejulio) [inline actions/platform/tam/setup_actions.py:17]: I suppose this is debug, lets remove or add more info :)

File: actions/platform/tam/tam_request.py [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+async def send_rbac_tam_request(tracker: Tracker, body: Dict[str, Any]):
+    if app.is_running_locally:
+        print("called send_rbac_tam_request in local envionment with body: ", body)
+
+        from unittest.mock import Mock
+
+        mock_response = Mock()
+        mock_response.ok = True
+
+        return mock_response
```
Comment (josejulio) [inline actions/platform/tam/tam_request.py:196]: This is interesting!

File: actions/platform/tam/tam_request.py [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+        "target_org": org_id,
+        "start_date": start_date,
+        "end_date": end_date,
+        "roles": roles,
+    }
+
+
+async def send_rbac_tam_request(tracker: Tracker, body: Dict[str, Any]):
+    if app.is_running_locally:
+        print("called send_rbac_tam_request in local envionment with body: ", body)
```
Comment (josejulio) [inline actions/platform/tam/tam_request.py:189]: Lets use the logger instead

File: actions/platform/tam/tam_request.py (hunk 2 above)
Comment (justinorringer) [reply actions/platform/tam/tam_request.py:196]: Google helped! I was trying to copy the access request flow but also check the status :)

File: actions/platform/tam/setup_actions.py (hunk 1 above)
Comment (justinorringer) [reply actions/platform/tam/setup_actions.py]: Yup, it's gone

**Review by josejulio — APPROVED** [review]
lgtm

**Pull request merged**